docs/conf/portland/2022/sponsors/prospectus.rst

```diff
@@ -0,0 +1,267 @@
+:template: {{year}}/generic.html
+:banner: _static/conf/images/headers/{{ shortcode }}-group.png
+
+Online Sponsorship Prospectus
+#############################
+
+.. contents:: Sections
+   :local:
+   :depth: 1
+   :backlinks: none
+
+Introduction
+============
+
+Welcome to our **2022 Conference Sponsorship Prospectus**.
+This document has been adapted for our hybrid conference format,
+including new benefits and increased value across our community.
+Our new hybrid format has brought many questions,
+and we're working to create innovative solutions where possible.
+
+We're excited to work with the organizations in our community to build the best possible event in {{ year }}.
+In particular, we would love your feedback on sponsorship levels and benefits.
+We expanded the offerings and introduced a lot of new things,
+but please let us know if there are other points of interaction with our community that would be valuable for you.
+
+Concept
+=======
+
+{% include "conf/sponsorship/concept.rst" %}
+
+Demographics
+============
+
+{% include "conf/sponsorship/demographics.rst" %}
+
+Why Sponsor
+===========
+
+{% include "conf/sponsorship/why.rst" %}
+
+Sponsorship Packages
+====================
+
+The following options are suggested sponsorship levels. We are happy to discuss adjustments and custom packages.
+
+Second Draft
+------------
+
+The **Second Draft** package is great for companies looking to hire or to promote a product.
+
+- Two (2) tickets
+- A small table at our **job fair**
```

Comment:
We might want to be careful with our phrasing around the job fair, in case we decide to have it virtual only. I don't have a great suggestion yet...